stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -318,35 +306,44 @@ extension ${Self}: BinaryFloatingPoint {
   // Internal implementation details of x86 Float80
   @_fixed_layout // FIXME(sil-serialize-all)
   @_versioned // FIXME(sil-serialize-all)
-  internal struct _Float80Representation {
+  internal struct _Representation {
     @_versioned // FIXME(sil-serialize-all)
-    internal var explicitSignificand: UInt64
+    internal var _storage: (UInt64, UInt16, /* pad: */ UInt16, UInt16, UInt16)
+
+    @_inlineable // FIXME(sil-serialize-all)
     @_versioned // FIXME(sil-serialize-all)
-    internal var signAndExponent: UInt16
+    @_transparent
+    internal var explicitSignificand: UInt64 { return _storage.0 }
+
+    @_inlineable // FIXME(sil-serialize-all)
     @_versioned // FIXME(sil-serialize-all)
-    internal var _padding: (UInt16, UInt16, UInt16) = (0, 0, 0)
+    @_transparent
+    internal var signAndExponent: UInt16 { return _storage.1 }
+
     @_inlineable // FIXME(sil-serialize-all)
     @_versioned // FIXME(sil-serialize-all)
+    @_transparent
     internal var sign: FloatingPointSign {
-      return FloatingPointSign(rawValue: Int(signAndExponent &>> 15))!
+      return FloatingPointSign(rawValue: Int(_storage.1 &>> 15))!
```

Comment:
Sure, we could. My personal style is generally to avoid invoking a computed property, all else being equal, but the clarity win of doing that is offset here by the opaqueness of the name.

Reply:
It seems to me that the only point of having these computed properties is that they're a meaningful abstraction over an otherwise opaque name.

Reply:
Fair.